src/docfx/lib/markdown/ExtractTitle.cs

```diff
@@ -16,7 +18,7 @@ public static MarkdownPipelineBuilder UseExtractTitle(this MarkdownPipelineBuild
         {
             return builder.Use(document =>
             {
-                var h1 = document.SkipWhile(block => block is YamlFrontMatterBlock).FirstOrDefault() as HeadingBlock;
+                var h1 = GetFirstAfterYamlNoneCommentHeadingBlock(document);
```

Comment:
how about `var h1 = document.SkipWhile(block => block is YamlBlock || IsComment(block)).FirstOrDefault() as HeadingBlock`

Reply:
I want there is just one YAMLBlock


---
In reply to: [210175302](https://github.com/dotnet/docfx/pull/3192#discussion_r210175302) [](ancestors = 210175302)